xlators/performance/open-behind/src/open-behind.c

```diff
@@ -479,7 +479,7 @@ ob_open(call_frame_t *frame, xlator_t *this, loc_t *loc, int flags, fd_t *fd,
         if (open_frame != NULL) {
             stub = fop_open_stub(open_frame, ob_open_resume, loc, flags, fd,
                                  xdata);
-            if (stub != NULL) {
+            if (stub != NULL && ob_inode != NULL) {
```

Comment:
This is not necessary. It will never be NULL here.

However, if it could be NULL, then we would leak `stub` and it would still crash in `ob_open_completed()` at line 499.